Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/ListView.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Xamarin.Forms.PlatformConfiguration.iOSSpecific
+{
+	using FormsElement = Forms.ListView;
+
+	public static class ListView
+	{
+		public static readonly BindableProperty BouncesProperty = BindableProperty.Create(nameof(Bounces), typeof(bool), typeof(ListView), true);
```

Comment:
if it's boolean property, should be IsBouncing, or HasBounces, CanBounces, IsBouncesEnabled,....